addons/web_tour/models/ir_http.py

```diff
@@ -6,6 +6,5 @@ class Http(models.AbstractModel):
 
     def session_info(self):
         result = super(Http, self).session_info()
-        if result['is_superuser']:
-            result['web_tours'] = request.env['web_tour.tour'].get_consumed_tours()
```

Comment:
I am not sure about this, but it should at least be in a separated commit (with everything else which concerns the consumed tours loading)